configs/linux.config

```diff
@@ -1312,12 +1403,185 @@ CONFIG_INIT_STACK_NONE=y
 # end of Kernel hardening options
 # end of Security options
 
-# CONFIG_CRYPTO is not set
+CONFIG_CRYPTO=y
```

Comment:
Can you disable `CONFIG_CRYPTO`?

Reply:
No, I can't. It is selected by `CONFIG_EXT4_FS` and `CONFIG_BLOCK`